package.json

```diff
@@ -32,6 +32,7 @@
     "mkdirp": "^1.0.4",
     "prettier": "^2.6.2",
     "pretty-quick": "^3.1.3",
+    "private-group-spec-vectors": "github:ssbc/private-group-spec#add-classic-unbox-vectors",
```

Comment:
added this as an extra dep, to not have to update `private-group-spec` initially. let's see if we can update it without too many breaking changes

Reply:
it seems we weren't using the spec for anything, so merged the deps now

Reply:
oops lol i searched for the wrong thing. but tests seem to be passing anyway